src/assets/CloudinaryFile.ts

```diff
@@ -282,14 +359,7 @@ class CloudinaryFile {
       const safeURL = encodeURI(url)
         .replace(/\?/g, '%3F')
         .replace(/=/g, '%3D');
-
-      // urlConfig.analytics is true by default, has to be explicitly set to false to overwrite
-      // Don't add analytics when publicId includes a '?' to not risk changing existing query params
-      if (this.urlConfig.analytics !== false && !(publicID.includes('?'))) {
-        return `${safeURL}?_a=${getSDKAnalyticsSignature(trackedAnalytics)}`;
-      } else {
-        return safeURL;
-      }
+      return this.appendQueryParamsToURL(safeURL, trackedAnalytics);
```

Comment:
This is a bug I think we have that I didn't want to fix with this PR.

The checks are relevant because we should not be replacing `?` with `%3F`, and that should be fixed promptly. 

So yes, the checks are meaningless now, however once we fix the bug we'll need to re-add them.